libs/error_injector/celix_properties/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+add_library(properties_ei STATIC src/celix_properties_ei.cc)
```

Comment:
Is there a reason why properties_ei and malloc_ei cannot be part of the error_injector library?

I understand that the current error_injector (interface) library provides the mechanisme to create error injection for functions, but I think we if we use error injection a single linkage setup is a bit user friendly.